packages/app/src/systems/Chains/eth/containers/EthAssetsDialog.tsx

```diff
@@ -49,4 +189,12 @@ const styles = {
   actionButton: cssObj({
     width: '100%',
   }),
+  dialogFooter: cssObj({
+    borderTop: '1px solid $intentsBase8',
+    justifyContent: 'center',
+    paddingTop: '$4',
+  }),
+  text: cssObj({
```

Comment:
avoid so generic names like `text`... everything can be a text

```suggestion
  headerInput: cssObj({
```